src/main/kotlin/org/kamiblue/client/module/AbstractModule.kt

```diff
@@ -33,9 +35,11 @@ abstract class AbstractModule(
     private val enabled = BooleanSetting("Enabled", false, { false }).also(::addSetting)
     private val visible = BooleanSetting("Visible", showOnArray).also(::addSetting)
     private val default = BooleanSetting("Default", false, { settingList.isNotEmpty() }).also(::addSetting)
+    val priorityForGui = IntegerSetting("Priority In GUI", 0, IntRange(0, 1000), 1, { ClickGUI.sortBy.value == ClickGUI.SortByOptions.CUSTOM }).also(::addSetting)
```

Comment:
Would be better to not have small step value for setting with big range. Set the step to 50, and fineStep 1 I think.